### PR TITLE
Newsletter-254-Translate-into-French

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/2023-05-17-policy.md [hunk 1]
```diff
@@ -43,7 +43,13 @@ h2:not(:first-of-type) { margin-top: 3em; }
 
 *Originally published in [Newsletter #253](/fr/newsletters/2023/05/31/#attente-de-la-confirmation-3--enchères-pour-lachat-dun-bloc-despace)*
 
-{% include specials/policy/fr/03-encheres-pour-l-achat-d-un-bloc-d-espace.md %}
+{% include specials/policy/fr/03-encheres-pour-lachat-dun-bloc-despace.md %}
```
Comment (bitschmidty) [inline _posts/fr/2023-05-17-policy.md:46]:
```suggestion
{% include specials/policy/fr/03-encheres-pour-l-achat-d-un-bloc-d-espace.md %}
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Comment by Copinmalin** [comment]
@bitschmidty : change seem not resolve problem :.-(

**Comment by Copinmalin** [comment]
@bitschmidty : hello, I need your help please

I don't understand why link for news 252 and 253 was good before add link for news 254 and are now in error

**Review by bitschmidty — APPROVED** [review]
Tested links, ACK

@Copinmalin let me know if you are ok with the changes I made.

**Comment by Copinmalin** [comment]
Thanks for your help @bitschmidty ,

I don't understand because two times I had no error in policy.md 
Capture d'écran 2023-06-26 191459
Capture d'écran 2023-06-26 191448

And after insert link for news 254, some error with previous link.
I'm so confused :-(

Many thanks for your patience with me

**Comment by Copinmalin** [comment]
@bitschmidty : we have a problem on this newsletter with a wrong link to policy : 
actually point to english version,

Need to be : 
{% include specials/policy/fr/04-estimation-taux-de-frais.md %}

new PR or can we change this one ?

**Comment by bitschmidty** [comment]
@Copinmalin can you open a new PR for the correction?

**Pull request merged**